tests/scanner/audit/gce_networks_rules_engine_test.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests the EnforcedNetworkRulesEngine."""
+
+import copy
+import itertools
+import mock
+import yaml
+
+from google.apputils import basetest
+from google.cloud.security.common.util import file_loader
+from google.cloud.security.scanner.audit.errors import InvalidRulesSchemaError
+from google.cloud.security.scanner.audit import base_rules_engine as bre
+from google.cloud.security.scanner.audit import gce_networks_rules_engine as gnre
+from google.cloud.security.scanner.audit import rules as scanner_rules
+from tests.unittest_utils import get_datafile_path
+
+
+# TODO: Define more tests
```

Comment:
It would be super awesome to add tests around the violation determination portion of the code.  One for when a violation is found.  One for when a violation is not found.